command_line/search_beam_position.py

```diff
@@ -153,43 +154,43 @@ def igrid(x):
     else:
         wide_search_offset = None
 
-    # DO A SIMPLEX MINIMIZATION
-
-    class test_simplex_method(object):
-        def __init__(selfOO, wide_search_offset=None):
-            selfOO.starting_simplex = []
-            selfOO.n = 2
-            selfOO.wide_search_offset = wide_search_offset
-            for ii in range(selfOO.n + 1):
-                selfOO.starting_simplex.append(flex.random_double(selfOO.n))
-            selfOO.optimizer = simplex_opt(
-                dimension=selfOO.n,
-                matrix=selfOO.starting_simplex,
-                evaluator=selfOO,
+    # Do a simplex minimization
+    class simplex_minimizer(object):
+        def __init__(self, wide_search_offset):
+            self.n = 2
+            self.wide_search_offset = wide_search_offset
+            self.starting_simplex = [  # guaranteed to be random
+                flex.double([0.11505456638977896, 0.6090665392794814]),
+                flex.double([0.13339096418598828, 0.24058961996534878]),
+                flex.double([0.3271390558111398, 0.8591374909485977]),
```

Comment:
I'm trying to understand why this change helps, if `flex.random_double` was indeed seeded correctly then how were we getting differences here on different test instances?

Reply:
tested this, and this was a red herring. I've put the random numbers back.